evrApp/Db/evrscale.db

```diff
@@ -7,6 +7,18 @@
 #  IDX = ID # corresponding to output mapping codes
 #  MAX = Maximum division
 
+record(bo, "$(P)PSPolarity-Sel")
+{
+  field( DTYP, "Obj Prop bool")
+  field( DESC, "Polarity of prescalers")
+  field( OUT , "@OBJ=$(EVR), PROP=PSPolarity")
```

Comment:
Should this EVR-wide setting be replicated for each prescalar?  Won't these records get out of sync?